src/opset/Conv_11.c

```diff
@@ -259,82 +170,78 @@ int Conv_{{op_version}}(connx_Graph* graph, __attribute__((unused)) uint32_t out
                     for (int32_t channel = 0; channel < channel_count; channel++) {
-                        _conv_{{ DTYPE | to_name }}(Y_flatten, X_flatten + channel * X_unit, feature_dim, feature_shape,
-                                            &x_iter, W_flatten + channel * W_unit, kernel_shape, dilations);
+
+                        //fprintf(stderr, "batch: %d, feature: %d, group: %d, channel: %d\n", batch, feature, g, channel);
+                        {{TYPE}}* X_array = X_flatten + X_unit * (channel_count * g + channel);
+                        {{TYPE}}* W_array = W_flatten + W_unit * (channel_count * feature + channel);
+
+                        /*
+                        fprintf(stderr, "\tX: ");
+                        for (int32_t i = 0; i < 3; i++) {
+                            fprintf(stderr, "%f ", X_array[i]);
+                        }
+                        fprintf(stderr, "\n");
+                        */
+                        /*
+                        fprintf(stderr, "\tW: ");
+                        for (int32_t i = 0; i < 3; i++) {
+                            fprintf(stderr, "%f ", W_array[i]);
+                        }
+                        fprintf(stderr, "\n");
+                        */
+
+                        conv_1d_{{DTYPE}}(Y_flatten, Y_shape + 2, X_array, feature_shape, W_array, kernel_shape, pads, dilations, strides);
                     }
 
                     if (B_flatten != NULL) {
-                        {{TYPE}} B_array[Y_unit];
-                        connx_{{ DTYPE | to_name }}_broadcast(Y_unit, B_array, 1, B_flatten);
-                        connx_{{ DTYPE | to_name }}_add(Y_unit, Y_flatten, Y_flatten, B_array);
-                        B_flatten++;
+                        {{TYPE}} bias = B_flatten[feature];
+                        //fprintf(stderr, "bias: %f\n", (float)bias);
+
+                        for (int32_t i = 0; i < Y_unit; i++) {
+                            Y_flatten[i] += bias;
+                        }
                     }
-                    */
-                    if (B_flatten != NULL) {
-                        B_flatten++;
+                    /*
+                    for (int32_t i = 0; i < Y_unit; i++) {
+                        fprintf(stderr, "%f ", Y_flatten[i]);
                     }
+                    fprintf(stderr, "\n");
+                    */
 
                     Y_flatten += Y_unit;
-                    W_flatten += W_feature_unit;
                 }
-
-                X_flatten += channel_count * X_unit;
+                X_flatten += X_unit * channel_count * group;
             }
-        }
 
-        // clang-format off
-        connx_Thread_run_all(run_{{DTYPE | to_name}}, work_count, works,
-                             sizeof(struct Parameter_{{DTYPE | to_name}}));
-        // clang-format on
+            break;
+        default:
+            connx_error("Conv: dimension is not supported yet: %d\n", feature_dim);
```

Comment:
For compatibility, leave original implementation and run at this point would be better. Especially, We don't have 2d+ on this PR

Reply:
2d and 3d is reimplemented!